### ISSUE TITLE
Cannot run new iOS app since latest upgrade

### ISSUE THREAD

**Issue by gdorbes** [apache/cordova-ios#1407]
# Bug Report

## Problem
Hello,

Since I have upgraded my iPhone to iOS 17.4 and my Mac M1 to Sonoma 14.4 with Xcode 15.3, I not able any longer to run new version of my app on iOS. Nothing else has changed about my app except these elements.

Deployment and build work fine, but when I try to run the app either on a simulator or a real device, the app is stopped and Xcode console returns the following messages:
```
Failed to request allowed query parameters from WebPrivacy.
Updated list with error: DownloadFailed.
```
I did not found details about these messages.

Could you help me?

Thanks

## Checklist

- [x] I searched for existing GitHub issues
- [x] I updated all Cordova tooling to most recent version
- [x] I included all the necessary information above

**Comment by SevenSystems** [comment]
I'm getting this in a WKWebView-based app (not Cordova, but in-house framework) on Catalyst. The app seems to work fine, but it does look worrying.

**Comment by gdorbes** [comment]
My bad: the issue was elsewhere

**Comment by arlo1990** [comment]
@gdorbes what was the issue and how did you resolve?

**Comment by TejasSatpute077** [comment]
@gdorbes 

I'm also encountering the same issue. Could you please explain how you resolved it in detail?

**Comment by Jakub-Horacek** [comment]
@gdorbes I have the same Issue, could you please provide more detail how have you resolved this issue? I would really appreciate it.

![image](https://github.com/apache/cordova-ios/assets/85481201/f56104fd-7f1d-47fe-a3a4-36875c2a0526)

**Comment by Jakub-Horacek** [comment]
@TejasSatpute077 Yes, I can try to provide as many details as you need, however I am really newbie iOS developer, so I am not sure what details I need to provide you.

When I launch my app in the simulator, it is stuck on the Launch Screen and the rest of the application is not displayed. There are no errors in the XCode Log. Only the warning mentioned in my previous message.

My app is using Ionic, Cordova, Capacitor and React.

**Comment by TejasSatpute077** [comment]
@Jakub-Horacek if possible, provide me a screenshot of the page

**Comment by Jakub-Horacek** [comment]
@TejasSatpute077 It is just a blank screen with app logo.
![image](https://github.com/apache/cordova-ios/assets/85481201/63bc791e-3239-484b-baa5-04734b54df81)

**Comment by TejasSatpute077** [comment]
@Jakub-Horacek in your config.xml file set AutoHideSplashScreen as true and clean and build project again

**Comment by Jakub-Horacek** [comment]
@TejasSatpute077 When I set the AutoHideSplashScreen to true, the result is same, but without the logo. 😕
![image](https://github.com/apache/cordova-ios/assets/85481201/1a70cee1-6546-4126-964b-cfc548d0e6da)

**Comment by TejasSatpute077** [comment]
@Jakub-Horacek in your package.json check required node version mentioned in
 "engines": {
    "node": ">=18.18.2 <19"
  },

and check which version of node you are using on terminal

node -v

and also check cordova version 

cordova --version

and

cordova platform ls

**Comment by Jakub-Horacek** [comment]
@TejasSatpute077 

There is no
```json
"engines": {
        "node": ">=18.18.2 <19"
      }
```
in my package.json, however it is MANY times (thousands) in the package.lock.json and each time with different `node` value.
I am currently sing the `v20.11.0` node version in my terminal.

I found in my package.json only this information about the cordova: `"cordova-plugin-ionic": "5.5.3",`
and I am using the `12.0.0 (cordova-lib@12.0.1)` in my terminal.

**Comment by TejasSatpute077** [comment]
@Jakub-Horacek 

I encountered the same problem you're facing and resolved it by starting fresh with a new setup. Before that, I tried various solutions for the white screen issue and warnings you're experiencing, but none worked. You might want to try a fresh setup, ensuring compatibility with your current Node.js version.

**Comment by Jakub-Horacek** [comment]
@TejasSatpute077 Okay, thank you a lot for your time.

**Comment by Jakub-Horacek** [comment]
@TejasSatpute077 So I managed to successfully connect to the Safari inspector for the JSContent as @breautek mentioned in the message above.

And I have these errors and warnings in the inspector:
![Screenshot 2024-04-30 at 16 51 05](https://github.com/apache/cordova-ios/assets/85481201/b5546413-97fe-4f5c-a1db-1eb64995a9e2)

The App is still not working but now I have some more informations, thank you.
